### PR TITLE
Make the build agent image name configurable

### PR DESCRIPTION
**Pull request by msteinert**
This should simplify things for people who want to hack on drone-exec.

### CODE REVIEW

File: engine/util.go [hunk 1]
```diff
@@ -33,3 +34,11 @@ func encodeToLegacyFormat(t *Task) ([]byte, error) {
 	// }
 	return json.Marshal(t)
 }
+
+func getenvDefault(varname, value string) string {
```
Comment (tboerger) [inline engine/util.go]: getEnvDefault :)

**Comment by tboerger** [comment]
Maybe you want to add it for entrypoint as well?

Beside that a small NITPIK

File: engine/util.go (hunk 1 above)
Comment (msteinert) [inline engine/util.go]: I was following the style of the Go standard library:

https://golang.org/pkg/os/#Getenv

**Comment by msteinert** [comment]
@tboerger I was thinking about entrypoint as well... But the implementation is a bit more complicated than this since it's a list. Maybe in a follow-on PR?

Without too much though, perhaps it could be a semi-colon separated list?

**Comment by tboerger** [comment]
These were just my 2 cents... Maybe @bradrydzewski thinks differently about it

**Comment by msteinert** [comment]
I just noticed this:

https://github.com/drone/drone/issues/1324

... plus some problems with this implementation. I'll take another crack at it later...

**Pull request closed without merging**